.github/workflows/cicd.yaml

```diff
@@ -166,6 +167,7 @@ jobs:
       - name: start services
         run: |
           sh steps/ut_zookeeper.sh start
+          sh steps/download_openmldb_spark.sh $SPARK_HOME
```

Comment:
should be ${{ env.SPARK_HOME }}
in a workflow runner, the shell don't know the $SPARK_HOME variable

Reply:
It is the error of `coverage` which lacks of `SPARK_HOME`. This `$SPARK_HOME` works for `cicd.yaml` workflow.

Reply:
I' wrong